golang/cosmos/x/swingset/config.go

```diff
@@ -31,17 +31,24 @@ const DefaultConfigTemplate = `
 # If relative, it is interpreted against the application home directory
 # (e.g., ~/.agoric).
 slogfile = "{{ .Swingset.SlogFile }}"
+
+# maxVatsOnline is the maximum number of vats that SwingSet kernel will bring online
+maxVatsOnline = {{ .Swingset.MaxVatsOnline }}
 `
 
 // SwingsetConfig defines configuration for the SwingSet VM.
 // TODO: Consider extensions from docs/env.md.
 type SwingsetConfig struct {
 	// SlogFile is the absolute path at which a SwingSet log "slog" file should be written.
 	SlogFile string `mapstructure:"slogfile" json:"slogfile,omitempty"`
+	// MaxVatsOnline is the maximum number of vats that the SwingSet kernel will have online
+	// at any given time.
+	MaxVatsOnline int `mapstructure:"maxVatsOnline" json:"maxVatsOnline,omitempty"`
```

Comment:
The toml cosmos config uses snake_case instead of camelCase. 

```suggestion
	MaxVatsOnline int `mapstructure:"max_vats_online" json:"maxVatsOnline,omitempty"`
```